### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by e154**
with a large number of configuration files in the directory dev, staging, prod, convenient to use a single configuration file

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -180,6 +182,15 @@ func init() {
 	}
 
 	appConfigPath = filepath.Join(AppPath, "conf", "app.conf")
+	commonConfigPath := filepath.Join(AppPath, "conf", "common.conf")
+	if utils.FileExists(commonConfigPath) {
+		CommonConfig, err := newAppConfig(appConfigProvider, commonConfigPath)
```
Comment (ysqi) [inline config.go]: Why not offter the test case ? CommonConfig is local variables  , Not use `:=`

**Comment by JessonChan** [comment]
This is not a very good idea in my opinion.

**Comment by ysqi** [comment]
hey @e154  ,Why need  common.conf  ?  You can use config item  `include` to load config from other fiel. such as like this:
```ini
AppName = crm
include "/var/app/crm/crm.config"
```

**Comment by e154** [comment]
Hi friends

: = - Yes it is a typo

This patch allows you to use separate directory configurations.
if you can use this architecture by standard means, I'll be glad to hear how to do it

config

**Pull request closed without merging**